features/images.feature

```diff
@@ -0,0 +1,40 @@
+@images
+Feature: OpenStack Image management
+  As an administrator
+  I want to make sure that OpenStack Image management component
+  is working correctly
+
+  @create_kvm
+  Scenario: Creation of KVM image
+    Given KVM source image exists
+    And glance image does not exist
+    When I create new glance image based on jeos
+    Then this image has non-empty ID
+    And its ID can be used to show the image info
+    And the status of the image is active
+    When I delete the image
```

Comment:
@jsuchome Based on [my discussion with @vmoravec](https://github.com/SUSE-Cloud/cct/pull/2#discussion_r29770123), we are proposing that for each of these 3 scenarios, you can delete the `When I delete the image` / `Then it is no longer listed` steps since they are already covered by `And glance image does not exist`.  But since this is the same for all three scenarios, you could extract it out into a [background](https://github.com/cucumber/cucumber/wiki/Background), e.g.:

```gherkin
Background:
    Given glance image does not exist
    And image quota has at least 2GB remaining
```

Of course you still need to test that an image is no longer listed after deletion, but this should be done via a dedicated scenario:

```gherkin
Scenario: Deletion of a KVM image
  Given a glance image based on jeos
  When I delete the image
  Then it is no longer listed
```